nbdev/processors.py

```diff
@@ -59,8 +59,10 @@ def _show_docs(trees):
     return [t for t in trees if isinstance(t,ast.Expr) and nested_attr(t, 'value.func.id')=='show_doc']
 
 def cell_lang(cell): return nested_attr(cell, 'metadata.language', 'python')
+
+def _want_hide(c): return 'hide' in c.directives_ or c.directives_.get('include:') == ['false']
```

Comment:
I found it odd that the colon is included in the key `include:` vs `include`.  Looking at the code this seems deliberate and is even in the tests for `processor.extract_directives`.

Reply:
I'm not sure why it wouldn't include the `:`? That's the thing that let's you know it's a key:value thing.

BTW why did you abstract out `_want_hide` here? Seems like you could easily inline it since you're only using it in one place?

Reply:
Would you mind taking a look at the new way I just pushed?  Just want to check if that is more in line with what you would do